include/functions.inc.php

```diff
@@ -1119,14 +1121,16 @@ function serendipity_addCategory($name, $desc, $authorid, $icon, $parentid) {
  * @param   int     The new category subcat hiding
  * @return null
  */
-function serendipity_updateCategory($cid, $name, $desc, $authorid, $icon, $parentid, $sort_order = 0, $hide_sub = 0, $admin_category = '') {
+function serendipity_updateCategory($cid, $name, $desc, $authorid, $parentid, $icon, $xmlimage, $mailimage, $sort_order = 0, $hide_sub = 0, $admin_category = '') {
```

Comment:
Honestly unsure about this. xmlimage looks to me like something that belongs into the sidebar plugin?